### ISSUE TITLE
Document export and processing telemetry data

### ISSUE THREAD

**Issue by mrjones-plip** [medic/cht-docs#311]
## Overview 
With the Sep 11 2020 go live of [CMoHM deployment](https://github.com/medic/config-moh-mali/tree/master/covid-surveillance) (private repo) we were trying to watch the [known performance issue](https://github.com/medic/config-moh-mali/issues/56) (again, private repo) and it came up that there's no documented process in the [docs site](https://docs.communityhealthtoolkit.org/) for how to get the telemetry data out CouchDB for use in external apps/scripts.  Further, it might be nice to have some prototypical usage of the resulting JSON file.

We do have two pages in the docs site related to this:
* where the telemetry data [is in CouchDB](https://docs.communityhealthtoolkit.org/apps/guides/database/#medic-user-username-meta)
* the [keys and what they mean](https://docs.communityhealthtoolkit.org/apps/guides/performance/telemetry/) in the resulting JSON data

## Related

It should be noted that the currently in use script ([`get_users_meta_docs.js`](https://github.com/medic/cht-core/blob/master/scripts/get_users_meta_docs.js)) is looking to be deprecated by exporting the data via couch2pg:

* [Add ability to replicate medic-users-meta database to PostgreSQL](https://github.com/medic/medic-infrastructure/issues/180)
* [Telemetry data not synced to postgres (private repo)](https://github.com/medic/medic-infrastructure/issues/180)
* [Analyse partners telemetry closely](https://github.com/medic/cht-core/issues/5993)

If any of these efforts are going to come to fruition in weeks not months, we may just want to wait and then 

## tl;dr
we should add below info, likely to [the telemetry page](https://docs.communityhealthtoolkit.org/apps/guides/performance/telemetry/) or to a new page linked from there.  The `plot_fp_time.py` is not publicly available (taken from private [medic-impact repo](https://github.com/medic/medic-impact/blob/master/learning_roadmap/telemetry/270_family_planning_visit_duration/README.md)), so we should either remove this reference or write a simple `.py` script to get folks started:

> To get telemetry data for a client, you will need to use [this script](https://github.com/medic/cht-core/blob/master/scripts/get_users_meta_docs.js) from the [cht-core](https://github.com/medic/cht-core/) repository. To run it from the root directory of the repo, use the following command, filling in the admin credentials and server address for the deployment:
> 
> ```
> $ node scripts/get_users_meta_docs.js --mode interactive --type telemetry https://USERNAME:PASSWORD@APPLICATION.SERVER/medic-users-meta
> ```
> 
> You will need to have node installed. If you get any error messages about missing packages, you can install them using
> 
> ```
> $ npm install PACKAGE_NAME
> ```
> 
> When prompted, select `Save all in working directory` then `Quit`. This will save a file named `telemetry.json` containing all of the telemetry data from a given deployment to a file in the `scripts` directory.
> 
> Move that file into this directory in the `medic-impact` repo and give it a prefix with the project name (e.g., name it `muso_telemetry.json`).
> 
> Then, run the [plot_fp_time.py](plot_fp_time.py) file in this directory.
> 
> ```
> $ python plot_fp_time.py
> ```
> 
> You will need to be using Python v3.6+ and have pandas and matplotlib installed.

FYI @abbyad , @sacul-git , @kennsippell, @n-orlowski  & @ecsalomon
